napari/utils/key_bindings.py

```diff
@@ -56,7 +56,10 @@ def hello_world(viewer):
 ]
 
 # global user keymap; to be made public later in refactoring process
-USER_KEYMAP: Mapping[str, Callable] = {}
+USER_KEYMAP: Mapping[KeyBinding, Callable] = {}
+
+# global plugin keymap; to be made public later in refactoring process
+PLUGIN_KEYMAP: Mapping[KeyBinding, Callable] = {}
```

Comment:
I do not understand idea of this dict. Why are they marked to be a part of public API?

Reply:
it's no more part of the public API than the other variables in here

as a rule of thumb i would not consider most things beyond the top two levels of the package to be truly public (so if it is not imported directly into `napari.utils` we need not worry about it)

Reply:
Bu there is a comment above that it will be public after refactor. And I do not understand why.

Reply:
Agreed, would like some clarification here on whether this was originally designed to be made public for some reason. I don't mind otherwise if it's marked with an underscore or not, I would tend to agree with Kira that it's not a huge danger, but could be convinced either way.